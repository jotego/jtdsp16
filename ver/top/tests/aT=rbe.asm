#Load accumulator from register
rb=0x10
a0=rb
re=0x20
a1=re
end:
goto end
