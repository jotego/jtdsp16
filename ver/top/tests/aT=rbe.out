r0=0x0000
r1=0x0000
r2=0x0000
r3=0x0000
rb=0x0010
re=0x0020
j =0x0000
k =0x0000
pr=0x0000
pt=0x0000
i =0x0000
a0=0x000100000
a1=0x000200000
x=0x0000
y=0x00000000
p=0x00000000
c0=0x0000
c1=0x0000
c2=0x0000
auc=0x0000
psw=0x0000
